cmd/query/app/flags_test.go

```diff
@@ -28,7 +28,7 @@ import (
 	spanstore_mocks "github.com/jaegertracing/jaeger/storage/spanstore/mocks"
 )
 
-func TestQueryBuilderFlags(t *testing.T) {
+func TestQueryBuilderFlagsDeprecation(t *testing.T) {
```

Comment:
since this test is call "test deprecation", it should only include deprecated flags. All other flags are already tested in the following test, there's no reason to repeat then here.

Reply:
refactored as suggested

Reply:
thx